core/arch/riscv64/mangle.c

```diff
@@ -32,6 +32,12 @@
 
 #include "../globals.h"
 #include "arch.h"
+#include "instr_create_shared.h"
+#include "instrument.h" /* instrlist_meta_preinsert */
+/* Make code more readable by shortening long lines.
+ * We mark everything we add as non-app instr.
+ */
+#define PRE instrlist_meta_preinsert
```

Comment:
I feel like seeing `PRE` down below isn't clear enough. Can you maybe make it more descriptive?

Reply:
You mean change `PRE` to `PREINSERT` or something? The `PRE` and `POST` naming convention is used everywhere, so I think it might be better to stay consistent.